symengine/ruby/ext/symengine/symengine.c

```diff
@@ -22,4 +24,11 @@ void Init_symengine() {
     rb_define_method(c_basic, "/", cbasic_div, 1);
     rb_define_method(c_basic, "**", cbasic_pow, 1);
     rb_define_method(c_basic, "-@", cbasic_neg, 0);
+    rb_define_method(c_basic, "to_s", cbasic_to_str, 0);
+    rb_define_method(c_basic, "expand", cbasic_expand, 0);
+
+    //Symbol class
+    c_symbol = rb_define_class_under(m_symengine, "Symbol", c_basic);
+    rb_define_alloc_func(c_symbol, csymbol_alloc);
```

Comment:
I just looked at how inheritance works and when you create a new `Symbol` instance both `cbasic_alloc` and `csymbol_alloc` gets called since `Symbol` is a child of `Basic`